### PR TITLE
PBS Bid Adapter: FPD typo fix

### PR DESCRIPTION
**Pull request by pm-harshad-mane**
## Type of change
- [ X ] Bugfix

## Description of change
gptPreAuction module sets adUnit.fpd.context.adServer with capital `S` in `adServer`
https://github.com/prebid/Prebid.js/blob/master/modules/gptPreAuction.js#L33
PBS code was reading it as `adserver` instead of `adServer`, Unit test cases were with the typo.
This PR fixes it.

### CODE REVIEW

**Review by bretg — APPROVED** [review]

**Review by jsnellbaker — APPROVED** [review]

**Comment by smenzer** [comment]
@mmoschovas have you had a chance to review this yet so we can get a 2nd approval before merging?

**Review by mmoschovas — APPROVED** [review]
LGTM. Please resolve conflicts so this can be merged

**Comment by pm-harshad-mane** [comment]
@jsnellbaker , @smenzer , @mmoschovas I see the changes in this PR are no more required as FPD2.0 related changes by @mmoschovas  has fixed the issue on 5th March, https://github.com/prebid/Prebid.js/commit/8c686a931022995265499034e5a6b6d0d2cea68d#diff-3375f581a781deb711719820144dc635c7852299259ba806911532c829349687

**Pull request closed without merging**
